test/test_faiss_and_milvus.py

```diff
@@ -506,7 +505,9 @@ def test_cosine_similarity(document_store):
         original_emb = indexed_docs[doc.content].astype("float32")
 
         # check if the stored embedding was normalized
-        assert np.allclose(original_emb, result_emb, rtol=0.2)  # high tolerance necessary for Milvus 2
+        np.testing.assert_allclose(
+            original_emb, result_emb, rtol=0.2, atol=5e-07
+        )  # high tolerance necessary for Milvus 2
```

Comment:
This change comes from the other PR, is it? #2375

Reply:
Yes, sorry for that! merging master into this branch should make it disappear.